samples/test/transcoder.test.js

```diff
@@ -0,0 +1,269 @@
+/**
+ * Copyright 2020, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const assert = require('assert');
+const {execSync} = require('child_process');
+
+const {Storage} = require('@google-cloud/storage');
+const bucketName = 'nodejs-samples-transcoder-test';
+const storage = new Storage();
+
+const projectId = process.env.GOOGLE_CLOUD_PROJECT;
+const projectNumber = process.env.GOOGLE_CLOUD_PROJECT_NUMBER;
+const location = 'us-central1';
+const templateId = 'my-nodejs-test-template';
+const preset = 'preset/web-hd';
+const templateName = `projects/${projectNumber}/locations/${location}/jobTemplates/${templateId}`;
+
+const testFileName = 'ChromeCast.mp4';
+const inputUri = `gs://${bucketName}/${testFileName}`;
+const outputUriForPreset = `gs://${bucketName}/test-output-preset/`;
+const outputUriForTemplate = `gs://${bucketName}/test-output-template/`;
+const outputUriForAdHoc = `gs://${bucketName}/test-output-adhoc/`;
+
+const cwd = path.join(__dirname, '..');
+const resourceFile = `testdata/${testFileName}`;
+
+before(async () => {
+  assert(
```

Comment:
Let's instantiate a client like so instead of requiring env variables: 

```
const client = new ClientName();
describe('deid', () => {
  let projectId;

  before(async () => {
    projectId = await client.getProjectId();
  });
```

Reply:
Done